test/components/views/spaces/SpacePanel-test.tsx

```diff
@@ -96,7 +96,13 @@ jest.mock("../../../../src/stores/spaces/SpaceStore", () => {
     const EventEmitter = require("events");
     class MockSpaceStore extends EventEmitter {
         invitedSpaces: SpaceKey[] = [];
-        enabledMetaSpaces: MetaSpace[] = [];
+        enabledMetaSpaces: MetaSpace[] = [
+            "home-space",
+            "favourites-space",
+            "people-space",
+            "orphans-space",
+            "video-rooms-space",
+        ] as MetaSpace[];
```

Comment:
Why not just use the enum values instead of a cast?

Reply:
Of course. Dont know why I copied over the strings.
I definitly want to use the enum here!

Reply:
I remember now why I used the raw string. It was because its inside a mock block. So we dont have any imports yet.
I changed it so we set the value in beforeEach